starwars_api/models.py

```diff
@@ -90,9 +86,13 @@ def __repr__(self):
 
 class FilmsQuerySet(BaseQuerySet):
     RESOURCE_NAME = 'films'
+    constructor = Films
 
     def __init__(self):
         super(FilmsQuerySet, self).__init__()
 
     def __repr__(self):
         return 'FilmsQuerySet: {0} objects'.format(str(len(self.objects)))
+
+resources = {'people':api_client.get_people, 'films':api_client.get_films}
```

Comment:
You're right, there're several things I could have done to make this cleaner, in retrospect. Had to use something like dictionaries for some of it, though. And the resources dictionary was inspired by the fact of the `RESOURCE_NAME` attribute already there.